contracts/src/v0.8/functions/tests/v1_X/FunctionsTermsOfServiceAllowList.t.sol

```diff
@@ -510,4 +510,30 @@ contract FunctionsTermsOfServiceAllowList_GetBlockedSendersInRange is FunctionsR
     vm.expectRevert(TermsOfServiceAllowList.InvalidCalldata.selector);
     s_termsOfServiceAllowList.getBlockedSendersInRange(1, BlockedSendersCount + 1);
   }
+
+  function test_MigratePreviousAllowlist() public {
```

Comment:
Let's name this in the same pattern as other unit tests, which would be:
`test_MigratePreviousAllowlist_Success`